src/Layout/GameDashboardLayout/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react'
+import { Box, Stack, AppBar, Toolbar, IconButton, Tabs, Tab } from '@mui/material'
+
+import { Games as GamesComponent } from '../../../pages/Games'
+
+
+function samePageLinkNavigation(
+	event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+) {
+	if (
+		event.defaultPrevented ||
```

Comment:
should be break into smaller functions

Reply:
The code is so dirty, to check many conditions like this you should create a function to solve that